app/views/podcast_switcher/_dropdown.html.erb

```diff
@@ -0,0 +1,61 @@
+<div class="prx-podcast-switcher dropdown" data-controller="focus">
+
+  <button class="btn btn-white dropdown-toggle" type="button" id="podcast-switcher" data-bs-toggle="dropdown" aria-expanded="false" data-action="focus#focus">
+    <% if @podcast %>
+      <% if img = (@podcast.feed_image || @podcast.itunes_image) %>
+        <%= image_tag(img.url, size: 30, alt: img.alt_text, class: "me-2") %>
+      <% else %>
+        <span class="material-icons size-30 me-2" aria-label="hidden">square</span>
+      <% end %>
+      <em class="flex-grow-1 text-start"><%= @podcast.title %></em>
+      <span class="material-icons justify-self-end ms-2" aria-label="hidden">expand_more</span>
+    <% else %>
+      <span class="material-icons size-30 me-2" aria-label="hidden">square</span>
+      <em class="flex-grow-1 text-start"><%= t(".select") %></em>
+      <span class="material-icons justify-self-end ms-2" aria-label="hidden">expand_more</span>
+    <% end %>
+  </button>
+
+  <ul class="dropdown-menu p-2 shadow" aria-labelledby="podcast-switcher">
+    <%# turbo-stream search form %>
+    <li class="search-filter mb-2">
+      <%= form_with(url: main_app.podcast_switcher_path, html: {autocomplete: "off"}, data: {controller: "search"}) do |f| %>
+        <div class="form-floating">
+          <%= f.search_text_field :q, {}, data: {focus_target: "field"} %>
+          <%= f.label :q, t(".search") %>
+        </div>
+        <%= f.submit class: "d-none", data: {search_target: "submit"} %>
+      <% end %>
+    </li>
+
+    <%# lazy-loaded list of podcasts %>
+    <%= turbo_frame_tag "podcast-switcher-list", src: main_app.podcast_switcher_path, loading: "lazy" do %>
+      <li class="d-flex justify-content-center">
+        <div class="spinner-border text-primary m-5" role="status"></div>
```

Comment:
Shows a spinner while the list-of-5 lazy loads.  Saves us querying the DB for a hidden dropdown every page load.

Reply:
<img width="71" alt="Screen Shot 2023-02-02 at 10 19 30 AM" src="https://user-images.githubusercontent.com/329012/216381241-df39dfab-7224-4822-8a27-b4f072d947be.png">

Seeing some titles hanging off the edge here.